eaf.py

```diff
@@ -255,6 +255,14 @@ def execute_function(self, buffer_id, function_name):
             except AttributeError:
                 self.message_to_emacs("Cannot execute function: " + function_name)
 
+    @dbus.service.method(EAF_DBUS_NAME, in_signature="sss", out_signature="")
+    def execute_function_with_arg(self, buffer_id, function_name, arg):
+        if buffer_id in self.buffer_dict:
+            try:
+                self.buffer_dict[buffer_id].execute_function_with_arg(function_name, arg)
+            except AttributeError:
+                self.message_to_emacs("Cannot execute function: " + function_name + "with arg: " + arg)
```

Comment:
Same as above, I think we should try to reuse `execute_function`. 

In this case, make the argument `args` defaulted to an empty string, and with a similar hack we did in `store_emacs_var` function:
```
@dbus.service.method(EAF_DBUS_NAME, in_signature="s", out_signature="")
    def store_emacs_var(self, var_dict_string):
        for var_pair in var_dict_string.split(":"):
            (var_name, var_value) = var_pair.split(",")
```
So that we can call Python function with arbitrary number of arguments :)

Reply:
How would the signature have to be declared then:

    @dbus.service.method(EAF_DBUS_NAME, in_signature="s", out_signature="")

I thought to go with one arg and use a string that gets splitted, because I don't know if there is another way to declare multiple args.